modules/project-comments/client/views/partials/period-documents-list.html

```diff
@@ -46,7 +46,7 @@
           <div class="btn-group">
             <button class="btn icon-btn dropdown-toggle" type="button"
               popover-class="context-menu"
-              popover-placement="'auto'"
+              popover-placement="bottom-right auto"
```

Comment:
In all cases adding the single-quotes to the property broke the anticipated placement (bottom-right by default, falling back to auto when the drop-menu clipped the scrollable parent.